tests/problems/generic/test_fgw_problem.py

```diff
@@ -153,6 +143,37 @@ def test_set_xy(self, adata_time: AnnData, tag: Literal["cost_matrix", "kernel"]
         assert isinstance(problem[0, 1].xy.data_src, np.ndarray)
         assert problem[0, 1].xy.data_tgt is None
 
+    @pytest.mark.fast()
+    @pytest.mark.parametrize(
+        "cost",
```

Comment:
I think of the fixture returning a tuple, you can deconstruct it here as `cost,cost_type,kwargs` so that it's clearer in the test (also in other places).